src/test/java/io/tiledb/java/api/AttributeTest.java

```diff
@@ -108,4 +108,12 @@ public void testAttributeSetFillValue() throws Exception {
           a.getFillValue().getSecond());
     }
   }
+
+  @Test
+  public void testAttributeSetNullable() throws Exception {
+    try (Context ctx = new Context();
+        Attribute a = new Attribute(ctx, "a2", Datatype.TILEDB_FLOAT32)) {
+      a.setNullable((short) 0);
```

Comment:
there appears to be an indentation issue here

Reply:
I think that this is ok. If you look at it, the `Attribute a = ..` is inside the `try` parentheses (`try (Context ctx = ... Attribute a = ...)` similarly to an `if () {}` block. So that's why the `a.setNullable` (line 116) is positioned two tabs (`\t`) after the `try` (line 114).

What do you think?